### PR TITLE
Practice page

### PR DESCRIPTION
**Pull request by Narekmaralchyan**

### CODE REVIEW

File: src/pages/private/Practice/PracticeUpdateForm.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                logoUrl:formValues.logoUrl,
+            }
+            console.log(updatedData)
+              mutate(updatedData,{
+                  onSuccess:()=>{
+                      message.success('Data updated successfully!')
+                  },
+                  onError:()=>{
+                      message.error('Failed')
+                      form.resetFields()
```
Comment (vanahit1989) [inline src/pages/private/Practice/PracticeUpdateForm.tsx]: there is no need to reset fields

File: src/hooks/useUpdatePractice.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import {collection, doc} from "firebase/firestore";
+import {firestore} from "../firebase.ts";
+import { useFirestoreDocumentMutation } from "@react-query-firebase/firestore";
```
Comment (vanahit1989) [inline src/hooks/useUpdatePractice.tsx]: move hooks to firebase folder

File: src/pages/private/Practice/PracticeUpdateForm.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+import {FC} from "react";
+import useUpdatePractice from "../../../hooks/useUpdatePractice.tsx";
+
+type TPracticeUpdateFormProps = {
+    data:TPractice;
+    id:string
+}
+const PracticeUpdateForm:FC<TPracticeUpdateFormProps> = ({data,id})=>{
+    const {form} = useForm()
+    const {mutate,isLoading} = useUpdatePractice(id)
```
Comment (vanahit1989) [inline src/pages/private/Practice/PracticeUpdateForm.tsx:16]: you can pass this id in hook instead of pass it from business logic

Comment (Narekmaralchyan) [reply src/pages/private/Practice/PracticeUpdateForm.tsx:16]: I am unable to pass an undefined value of 'id' to the 'doc()' function when creating a document reference for a mutation hook.

File: src/hooks/useUpdatePractice.tsx (hunk 2 above)
Comment (Narekmaralchyan) [reply src/hooks/useUpdatePractice.tsx]: ok

File: src/pages/private/Practice/PracticeUpdateForm.tsx (hunk 1 above)
Comment (Narekmaralchyan) [reply src/pages/private/Practice/PracticeUpdateForm.tsx]: ok

**Review by vanahit1989 — APPROVED** [review]

**Pull request merged**